ui/src/ducks/app/monitoring.js

```diff
@@ -121,14 +128,31 @@ export function* fetchClusterStatus() {
     clusterHealth.apiServerStatus = getClusterQueryStatus(results[0]);
     clusterHealth.kubeSchedulerStatus = getClusterQueryStatus(results[1]);
     clusterHealth.kubeControllerManagerStatus = getClusterQueryStatus(
-      results[2]
+      results[2],
     );
     yield put(setPrometheusApiAvailable(true));
   } else {
+    const prometheusPod = yield call(
+      ApiK8s.getPodInNamespace,
+      'metalk8s-monitoring',
+      'prometheus',
```

Comment:
Is this supposed to be the _Pod_ `metadata.name`? If it is, this won't work, as the actual _Pod_ is created through a _StatefulSet_. If it isn't, then the `getPodInNamespace` method is confusing (but that's debt, not your PR).